corehq/motech/repeaters/models.py

```diff
@@ -1265,6 +1264,11 @@ class Meta:
             )
         ]
         constraints = [
+            models.UniqueConstraint(
```

Comment:
Why is this required? Can we just not just set 

```
couch_id = models.CharField(max_length=36, null=False, blank=False)
```
and make the index unique?

I believe indices should be more performant than constraints?